algorithms/ar-isrt/javascript/insertion_sort.js

```diff
@@ -0,0 +1,20 @@
+let insertionSort = (array) => {
+  for (outer = 1; outer < array.length; outer++) {
+    for (inner = 0; inner < outer; inner++) {
+      console.log(array.join(' '))
+      if (array[outer] < array[inner]) {
+        const [element] = array.splice(outer, 1)
+        array.splice(inner, 0, element)
```

Comment:
Codacy found an issue: ['inner' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6335885)